app/models/suggested_councillor.rb

```diff
@@ -1,5 +1,6 @@
 class SuggestedCouncillor < ActiveRecord::Base
   has_one :authority, through: :councillor_contribution
   belongs_to :councillor_contribution
-  validates :councillor_contribution, :name, :email, presence: true
+  validates :email, :name, presence: true
+  validates_email_format_of :email
```

Comment:
This should be tested on the model with the `validate_email_format_of` [method that the gem provides](https://github.com/validates-email-format-of/validates_email_format_of#usage). Something like:

```
  describe "must have a valid email attribute" do
    it { expect(SuggestedCouncillor.new).to validate_email_format_of :email }
  end
```